.github/continuous-integration.yml

```diff
@@ -0,0 +1,40 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  lint:
+    name: "Lint"
+
+    runs-on: "ubuntu-latest"
+
+    strategy:
+      matrix:
+        php-binary:
+          - "php7.4"
```

Comment:
Since I'm not sure whether we need any specific extensions not already installed, perhaps it's sufficient not to use [`shivammathur/setup-php`](https://github.com/shivammathur/setup-php) yet.